### PR TITLE
Adding docker file and changing readme

### PR DESCRIPTION
**Pull request by vikasshiva**
This is to fix the issue of adding a docker image to Prometheus beat.

### CODE REVIEW

**Comment by boernd** [comment]
Thx. I think the Dockerfile is missing in this PR? The commit just contains changes to the README.md

File: README.md [hunk 1]
```diff
@@ -152,9 +152,7 @@ cd ${GOPATH}/github.com/infonova/prometheusbeat
 git clone https://github.com/infonova/prometheusbeat
 ```
 
-
-For further development, check out the [beat developer guide](https://www.elastic.co/guide/en/beats/libbeat/current/new-beat.html).
-
+This will fetch and create all images required for the build process. The hole process to finish can take several minutes.
```
Comment (boernd) [inline README.md]: This line can be removed.

**Comment by boernd** [comment]
Thx @vikasshiva

**Comment by vikasshiva** [comment]
Hi,

Sorry for the inconvenience I checked in Dockerfile now

**Pull request merged**